README.md

```diff
@@ -37,6 +38,27 @@ Amber is packaged in the following distros:
 
 Arch (AUR) - `amber-bash-bin`
 
+#### Nix
+
+The package is already containing all the required install scripts and dependencies. You can use the flake as:
```

Comment:
Perhaps add here that amber-lang has been packaged upstream since https://github.com/NixOS/nixpkgs/pull/313774.